prep-html-syntax/index.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+  <title>  </title>
```

Comment:
Hi Kevin, be sure to put your `<title></title>` element inside of a `<head></head>` element.